apps/airpollution/airpollution.star

```diff
@@ -0,0 +1,290 @@
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+load("time.star", "time")
+
+### MAIN CODE ###
+
+def main(config):
+    status_code = 200
+
+    airly_apikey = config.get("api_key")
+    airly_id = config.get("airly_id")
+
+    airly_street_out_cache = cache.get("airly_street_in_cache")
```

Comment:
The cache module is scoped to the app, but not to individual installations of the app. That means that this cache key will be read and written to by all users of the app.

The best path around this problem is likely to rely on the http client's builtin cache functionality. Check out [this post](https://discuss.tidbyt.com/t/http-caching-client/5129) for more info!